src/powershell/en-US/FinOpsToolkit.strings.psd1

```diff
@@ -10,4 +10,7 @@ ConvertFrom-StringData -StringData @'
     NewDirectory = Creating directory '{0}'.
     TemplateNotFound = Could not find template 'main.bicep' at path '{0}'.
     ContextNotFound = Could not retrieve Az context. Run Az-Login.
-'@
\ No newline at end of file
+    DeleteCostExportFailed = Delete Cost Management export operation failed with message: '{0}'.
```

Comment:
Can we include an error code as well as the message? You can just append it to the end:
```suggestion
    DeleteCostExportFailed = Delete Cost Management export operation failed with message: '{0}' (Code: {1}).
```